tests/e2e/crash-before-createdb/07-assert.yaml

```diff
@@ -1,14 +1,4 @@
 apiVersion: v1
-kind: Event
```

Comment:
It is been  a while for me so could you tell me why we do not need this anymore?

Reply:
Because the install is so fast, we no longer need the Event for it.  So all of the events for install and uninstall were removed.